data/mods/Magiclysm/enchanted/enchanted_rings.json

```diff
@@ -0,0 +1,277 @@
+  },
+  {
+    "copy-from": "mring_gold",
+    "type": "GENERIC",
+    "id": "mring_intelligence_3",
+    "name": "ring of intelligence +3",
+    "name_plural": "rings of intelligence +3",
+    "description": "A golden ring that makes you a little stronger when you wear it.",
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "INTELLIGENCE", "add": 3 } ] } ] }
+  },
+  {
+    "copy-from": "mring_platinum",
+    "type": "GENERIC",
+    "id": "mring_intelligence_4",
+    "name": "ring of intelligence +4",
+    "name_plural": "rings of intelligence +4",
+    "description": "A platinum ring that makes you a little stronger when you wear it.",
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "INTELLIGENCE", "add": 4 } ] } ] }
+  },
+  {
+    "copy-from": "mring_copper",
+    "type": "GENERIC",
+    "id": "mring_perception_1",
+    "name": "ring of perception +1",
+    "name_plural": "rings of perception +1",
+    "description": "A copper ring that makes you a little stronger when you wear it.",
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "PERCEPTION", "add": 1 } ] } ] }
+  },
+  {
+    "copy-from": "mring_silver",
+    "type": "GENERIC",
+    "id": "mring_perception_2",
+    "name": "ring of perception +2",
+    "name_plural": "rings of perception +2",
+    "description": "A silver ring that makes you a little stronger when you wear it.",
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "PERCEPTION", "add": 2 } ] } ] }
+  },
+  {
+    "copy-from": "mring_gold",
+    "type": "GENERIC",
+    "id": "mring_perception_3",
+    "name": "ring of perception +3",
+    "name_plural": "rings of perception +3",
+    "description": "A golden ring that makes you a little stronger when you wear it.",
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "PERCEPTION", "add": 3 } ] } ] }
+  },
+  {
+    "copy-from": "mring_platinum",
+    "type": "GENERIC",
+    "id": "mring_perception_4",
+    "name": "ring of perception +4",
+    "name_plural": "rings of perception +4",
+    "description": "A platinum ring that makes you a little stronger when you wear it.",
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "PERCEPTION", "add": 4 } ] } ] }
+  },
+  {
+    "copy-from": "mring_copper",
+    "type": "GENERIC",
+    "id": "mring_speed_3",
+    "name": "ring of speed +3",
+    "name_plural": "rings of speed +3",
+    "description": "A copper ring that makes you a little stronger when you wear it.",
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "SPEED", "add": 3 } ] } ] }
```

Comment:
Is SPEED both movecost and attackcost?

Reply:
Yes, speed is a general figure for everything. Right now there's no attack_cost, when there is, I'll create attack_cost specific items.